test/views/declaration/spec/ViewMatchers.scala

```diff
@@ -248,10 +230,16 @@ trait ViewMatchers {
     )
 
   def haveFieldErrorLink(fieldName: String, link: String): Matcher[Element] =
-    new ContainElementWithIDMatcher(s"error-message-$fieldName-input") and new ElementHasAttributeValueMatcher(
-      "href",
-      link
-    )
+    new Matcher[Element] {
```

Comment:
I thinking having specialized class for this would be nice too - reduce of class creation

Reply:
yes, fixed